includes/settings.php

```diff
@@ -16,111 +16,199 @@
+ */
+function rc_pro_features() {
+	?>
+
+	<div class="rc-pro-features">
+
+		<hr>
+		<h4><?php _e( 'Need more control?' ); ?></h4>
+		<p><?php _e( "Take your membership site to the next level with Restrict Content Pro. With RCP, you can:", 'restrict-content' ); ?></p>
+		<p>
+		<ul class="rc-settings-list" style="list-style-type: disc; list-style-position: inside">
+			<li><?php _e( 'Charge for access and add a recurring revenue stream to your business. PayPal, Stripe, 2Checkout, Authorize.net, Braintree, and Manual Payments are all supported.', 'restrict-content' ); ?></li>
+			<li><?php _e( 'Send and receive emails when members sign up, renew, cancel, and expire.', 'restrict-content' ); ?></li>
+			<li><?php _e( 'Generate invoices for subscription payments.', 'restrict-content' ); ?></li>
+			<li><?php _e( 'Prevent account sharing.', 'restrict-content' ); ?></li>
+			<li><?php _e( 'Subscribe members to your Mailchimp, AWeber, ConvertKit, ActiveCampaign, Campaign Monitor, GetResponse, or MailPoet mailing lists.', 'restrict-content' ); ?></li>
+			<li><?php _e( 'Drip content on a schedule.', 'restrict-content' ); ?></li>
+			<li><?php _e( 'Offer group accounts to businesses and other organizations.', 'restrict-content' ); ?></li>
+			<li><?php _e( 'Sell websites using WordPress multisite.', 'restrict-content' ); ?></li>
+			<li><?php _e( '... and a lot more!', 'restrict-content' ); ?></li>
+		</ul>
+		<?php printf(
+			__( '<a href="%s" target="_blank" rel="noopener noreferrer">Find out more...</a>', 'restrict-content' ),
+			'https://restrictcontentpro.com/?utm_campaign=restrict-content&utm_medium=admin&utm_source=settings&utm_content=main'
+		); ?>
+		</p>
+
+	</div>
+
+	<?php
+}
+
+/**
+ * Display a sidebar element
+ */
+function rc_display_sidebar() {
+	?>
+
+	<div class="rc-settings-sidebar">
+
+		<div class="rc-settings-sidebar-content">
+
+			<div class="rc-sidebar-header-section">
+				<img class="rc-bcfm-header" src="<?php echo esc_url( RC_PLUGIN_URL . '/includes/assets/images/bfcm-header.svg' ); ?>">
+			</div>
+
+			<div class="rc-sidebar-description-section">
+				<p class="rc-sidebar-description"><?php _e( 'Save on all Restrict Content Pro purchases <strong>this week</strong>, including renewals and upgrades!', 'restrict-content' ); ?></p>
+			</div>
+
+			<div class="rc-sidebar-coupon-section">
+				<label for="rc-coupon-code"><?php _e( 'Use <strong>COUPON CODE</strong> at checkout:', 'restrict-content' ); ?></label>
+				<input id="rc-coupon-code" type="text" value="BFCM2019" readonly>
+				<p class="rc-coupon-note"><?php _e( 'The sale ends the 6th of December at midnight (UTC-6). Use the same coupon code to save on <a href="https://sandhillsdev.com/projects/" target="_blank">our other WordPress plugins</a>.', 'restrict-content' ); ?></p>
+			</div>
+
+			<div class="rc-sidebar-footer-section">
+				<a class="rc-cta-button" href="https://restrictcontentpro.com/pricing/"><?php _e( 'Upgrade Now!', 'restrict-content' ); ?></a>

```

Comment:
Do we want to add campaign data to this URL? The other URLs have them.

Reply:
Yep! Marketing is getting those together for me and I'll add them to all projects. I keep forgetting that I'm not the one who merges these PRs... so I should make sure y'all know that beforehand lol.

We can let this sit. Once I get the UTM goodies from marketing, I'll be adding it and then I'll re-request your review.

Reply:
@SDavisMedia  Sounds good! Other than that I'm happy with everything.